src/main/java/core/org/akaza/openclinica/domain/datamap/Item.java

```diff
@@ -168,6 +169,14 @@ public void setDescription(String description) {
 		this.description = description;
 	}
 
+    @Column(name = "brief_description")
+    public String getBriefDescription() {
```

Comment:
why are you adding a new column?

Reply:
If we need to add a new column, we need to create liquibase script. 
@jkeremian can you review this please?

Reply:
The column is already in the DB, just not exposed in the bean, so no need to liquibase
CMIIW